src/Phoenix/PhoenixClient.ts

```diff
@@ -26,43 +23,27 @@ export class PhoenixClient {
     provisionData: IProvosionData
   ): Promise<IPhoenixResponse<IPhoenixConnectionInfoResult>> {
     try {
-      const connectionStatus: ContainerConnectionInfo = {
-        status: ConnectionStatusType.Connecting,
-      };
-      useNotebook.getState().setConnectionInfo(connectionStatus);
-      const response = await window.fetch(`${this.getPhoenixContainerPoolingEndPoint()}/provision`, {
+      const response = await window.fetch(`${this.getPhoenixContainerPoolingEndPoint()}/allocate`, {
         method: "POST",
         headers: PhoenixClient.getHeaders(),
         body: JSON.stringify(provisionData),
       });
       let data: IPhoenixConnectionInfoResult;
       if (response.status === HttpStatusCodes.OK) {
         data = await response.json();
-        if (data && data.notebookServerUrl) {
-          connectionStatus.status = ConnectionStatusType.Connected;
-          useNotebook.getState().setConnectionInfo(connectionStatus);
-        }
-      } else {
-        connectionStatus.status = ConnectionStatusType.Failed;
-        useNotebook.getState().setConnectionInfo(connectionStatus);
       }
-
       return {
         status: response.status,
         data,
       };
     } catch (error) {
-      const connectionStatus: ContainerConnectionInfo = {
-        status: ConnectionStatusType.Failed,
-      };
-      useNotebook.getState().setConnectionInfo(connectionStatus);
       console.error(error);
       throw error;
     }
   }
 
   public static getPhoenixEndpoint(): string {
-    const phoenixEndpoint = userContext.features.junoEndpoint ?? configContext.JUNO_ENDPOINT;
+    const phoenixEndpoint = userContext.features.phoenixEndpoint ?? configContext.JUNO_ENDPOINT;
```

Comment:
maybe make this 
userContext.features.phoenixEndpoint ?? userContext.features.junoEndpoint ?? configContext.JUNO_ENDPOINT

Reply:
updated